src/js/personalization/dashboard-beta/containers/ClaimsAppealsWidget.jsx

```diff
@@ -78,6 +81,28 @@ class ClaimsAppealsWidget extends React.Component {
     return null;
   }
 
+  renderWidgetDowntimeNotification = (appName) => {
+    return (status, downtimeWindow, downtimeMap, children) => {
+      switch (status) {
+        case 'down':
+          return (
+            <div>
+              <AlertBox
+                content={<div>
+                  <h4 className="usa-alert-heading">{appName} is down for maintenance</h4>
+                  <p>We’re making some updates to our {appName.toLowerCase()} tool. We’re sorry it’s not working right now and hope to be finished by {downtimeWindow.startTime.format('MMMM Do')}, {downtimeWindow.endTime.format('LT')}. Please check back soon.</p>
+                </div>}
+                isVisible
+                status="warning"/>
+            </div>
+          );
+        default:
+          return children;
+      }
+    };
+  }
+
```

Comment:
It makes me so happy to see the first implementation of the custom render function.